gitbootcamp.rst

```diff
@@ -305,6 +293,26 @@ Pull requests can be accepted and merged by a Python Core Developer.
 3. Press the ``Confirm squash and merge`` button.
 
 
+Backporting Merged Changes
+--------------------------
+
+A pull request may need to be backported into one of the maintenance branches
+after it has been accepted and merged into ``master``.  It is usually indicated
+by the label ``needs backport to X.Y`` on the pull request itself.
+
+Use the utility script `cherry_picker.py <https://github.com/python/core-workflow/tree/master/cherry_picker>`_
+from the `core-workflow  <https://github.com/python/core-workflow>`_
+repository to backport the commit.
+
+The commit hash for backporting is the squashed commit that was merged to
+the ``master`` branch.  On the merged pull request, scroll to the bottom of the
+page.  Find the event that says something like::
```

Comment:
When I resolved conflict while cherry-picking to 3.6 branch, I want to use squash commit of 3.6 branch
for backporting to 3.5.

Reply:
When backporting the squashed commit from `master`, we get to keep the reference to the original PR on `master`, so bedevere-bot can remove the `needs backport to ..` labels.

Does this happen when backporting from 3.6 to 3.5?

Reply:
It should - the trick in this case is that the "needs backport to 3.5" label should go on the *backport* PR against the 3.6 branch, rather than the original PR against master.

Reply:
Right, the only time it makes sense if when the backport from master *fails*, and even then, it only applies for security fixes, or the times when we have two full maintenance branches open (i.e. from beta 1 of the upcoming stable through to the release of that branch and the subsequent final full maintenance release of the now previous stable branch).

However, now that we branch at beta, we're likely to spend around 1/3rd of the time in that state.

Reply:
@Mariatta Clarifying: I don't think the above needs to be covered in this PR, but there *are* cases where we want to do X.Y -> X.Y-1 backports, rather than backporting directly from master.

Reply:
Thanks :) I'm going to merge this.